.circleci/windows/msys2_install.bat

```diff
@@ -0,0 +1,35 @@
+@echo on
+IF "%WIN_MSYS2_ROOT%"=="" SET "WIN_MSYS2_ROOT=C:\tools\msys64"
+setx WIN_MSYS2_ROOT %WIN_MSYS2_ROOT%
+
+SET "_PATHS=%WIN_MSYS2_ROOT%\mingw64\bin;%WIN_MSYS2_ROOT%\usr\bin;%WIN_MSYS2_ROOT%"
+ECHO."%PATH%" | FINDSTR /V /C:"%_PATHS%">nul && SET "PATH=%_PATHS%;%PATH%"
+SET "_PATHS="
+
+SETLOCAL ENABLEEXTENSIONS
+
+IF EXIST "%WIN_MSYS2_ROOT%\mingw64" GOTO :MSYS2_UPDATE
+md "%WIN_MSYS2_ROOT%"
+pushd "%WIN_MSYS2_ROOT%\.."
+
+curl -s -O https://tukaani.org/xz/xz-5.2.4-windows.zip
```

Comment:
MSYS2 is distributed as this inadequate (for windows) .xz format which requires an additional tool to decompress it.
I didn't want to use 7z because it might not be available